example/lambdas/python-processing/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-buster
```

Comment:
Just curious- why Debian v11 over v10 or v12?

Reply:
that's a fair question: I don't remember why I did that and I'm pretty sure there was no reason, I'm switching it back

Reply:
I think it was me - I just found bullseye as an image with 3.10 and used it. I didn't know what buster or bullseye meant, so TIL.